src/api.ts

```diff
@@ -13,14 +13,15 @@ import { l } from './logger';
 import { pendingHashes } from './builder';
 import { exec } from 'child_process';
 
-import { CONTAINER_EXPIRY_TIME, START_TIME, TEN_MINUTES } from './constants';
+import { CONTAINER_EXPIRY_TIME } from './constants';
 import { timing } from './stats';
 
 type APIState = {
-	accesses: Map< CommitHash, number >;
+	accesses: Map< ContainerName, number >;
 	branchHashes: Map< CommitHash, BranchName >;
 	containers: Map< string, Docker.ContainerInfo >;
```

Comment:
Before this map contained only running containers, now it contain all containers present in the docker daemon.

The function that populates this map has been renamed from `refreshRunningContainers` to `refreshContainers` to reflect that fact, and all existing usages of `state.containers` have been refactored when needed to filter running containers.